draft-ietf-tls-exported-authenticator.md

```diff
@@ -142,70 +175,103 @@ CertificateVerify
        } CertificateVerify;
 
 The algorithm field specifies the signature algorithm used (see Section 4.2.3 of {{!TLS13}}
-for the definition of this field). The signature is a digital signature using that algorithm.
-The signature scheme MUST be a valid signature scheme for TLS 1.3.  This excludes all RSASSA-PKCS1-v1_5
-algorithms and ECDSA algorithms that are not supported in TLS 1.3.  For servers,
-this signature scheme must match one of the signature and hash algorithms advertised
-in the signature_algorithms extension of the ClientHello.  The signature is computed using the over the concatenation of:
+for the definition of this field).  The signature is a digital signature
+using that algorithm.  The signature scheme MUST be a valid signature
+scheme for TLS 1.3.  This excludes all RSASSA-PKCS1-v1_5 algorithms and
+ECDSA algorithms that are not supported in TLS 1.3.  If an authenticator
+request is present, the signature algorithm MUST be chosen from one of
+the signature schemes in the authenticator request.  Otherwise, the signature
+algorithm used should be chosen from the "signature_algorithms" extension
+of the ClientHello used in the connection handshake.
+
+The signature is computed using the over the concatenation of:
 
 * A string that consists of octet 32 (0x20) repeated 64 times
 * The context string "Exported Authenticator" (which is not NULL-terminated)
 * A single 0 byte which serves as the separator
-* The value
-Hash(Handshake Context || Certificate) where Hash is the hash function for the handshake.
+* If the authenticator request is present, the value
+Hash(Handshake Context || authenticator request || Certificate)
+* If an authenticator request is not present, the value
+Hash(Handshake Context || Certificate)
+where
+Hash is the hash function for the handshake.
```

Comment:
Right now, it's just text in-line.  Probably the simplest thing to do is `Hash(...)` (i.e. with backticks).  That renders as a code block in HTML and quotes in the text.

You can use triple-backticks to put it on a second line, but that doesn't appear to look good in the text version since it exits the bulleted list.

Reply:
I'll fix this up.